core/src/price_estimation.rs

```diff
@@ -61,6 +61,7 @@ impl PriceOracle {
             contract,
             token_data.clone().into(),
         ));
+        token_info_fetcher.cache_all(10).await?;
```

Comment:
Not really related to this PR and not a big issue but I didn't really get at first was this argument was. I guess it makes sense when in an IDE as it tells you what the argument name, but maybe we can use a reasonable default given there is only one call site right now.